.travis.yml

```diff
@@ -1,5 +1,4 @@
 sudo: false
-cache: bundler
```

Comment:
Then please tell me why travis is failing on `bundle install --deployment` and how it can be solved other than by checking the .lock files into git, which it would be silly to do without appraisal.